libs/hmpb/src/generate_fixtures.test.ts

```diff
@@ -27,7 +27,10 @@ async function expectToMatchSavedPdf(
     { page: actualPage },
     { page: expectedPage },
   ] of pdfPagePairs) {
-    expect(toImageBuffer(actualPage)).toMatchImage(toImageBuffer(expectedPage));
+    expect(toImageBuffer(actualPage)).toMatchImage(
+      toImageBuffer(expectedPage),
+      { dumpDiffToConsole: true }
+    );
```

Comment:
Used this for debugging image diffs, which are otherwise not visible at all, so leaving this in